tests/unit/services/phone-input-test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | phone-input', function(hooks) {
+  setupTest(hooks);
+
+  test('load is thenable on first and subsequent renders', async function(assert) {
+    assert.expect(2);
+
+    let service = this.owner.lookup('service:phone-input');
+
+    await service.load().then(() => {
```

Comment:
Can we use one syntax: async/await or Promises? I suspect this would work:
```suggestion
    service.load().then(() => {
```